### DISCUSSION TITLE
HEXA_mock & HEXA_Dcluster

### DISCUSSION THREAD

**Discussion by johnyquest7** [KeremTurgutlu/self_supervised#28] [Ideas]
@KeremTurgutlu have you tried HEXA_mock & HEXA_Dcluster?

**Comment by johnyquest7** [comment]
https://www.microsoft.com/en-us/research/blog/hexa-self-supervised-pretraining-with-hard-examples-improves-visual-representations/

**Reply by KeremTurgutlu** [reply]
Thanks for sharing! I will take a look at it.
